### ISSUE TITLE
Non-Zero Exit Status on Install Attempt

### ISSUE THREAD

**Issue by andtheWings** [ecohealthalliance/doltr#39]
I'm getting the following error when trying to install:

```
> remotes::install_github("ecohealthalliance/doltr")
Downloading GitHub repo ecohealthalliance/doltr@HEAD
✓  checking for file ‘/tmp/RtmprX8RPB/remotes574d3e6d6105/ecohealthalliance-doltr-48ccd34/DESCRIPTION’ ...
─  preparing ‘doltr’:
✓  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘doltr_0.0.0.9000.tar.gz’
   
Installing package into ‘/home/riggins/R/x86_64-pc-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
* installing *source* package ‘doltr’ ...
** using staged installation
** R
Error in parse(outFile) : 
  /tmp/RtmpoIRqNg/R.INSTALL5886c27cea38e/doltr/R/dolt-types.R:83:25: unexpected input
82:   if (!all(vapply(obj, is.raw, logical(1)))) "Stop only lists of raw vectors (blobs) allowed"
83:   nb <- max(vapply(obj, \
                            ^
ERROR: unable to collate and parse R files for package ‘doltr’
* removing ‘/home/riggins/R/x86_64-pc-linux-gnu-library/4.0/doltr’
Warning message:
In i.p(...) :
  installation of package ‘/tmp/RtmprX8RPB/file574d33e6b1a48/doltr_0.0.0.9000.tar.gz’ had non-zero exit status
```

**Comment by noamross** [comment]
I believe this was caused by use of the `\(x)` lambda syntax, introduced in R 4.1.0, as it looks like you are using 4.0.0.  I've removed it in https://github.com/ecohealthalliance/doltr/commit/112acb5cb79bf29908fd8af4ff00f90915abb43b, as it wasn't needed for this line.  Please let me know if this fixes your install issues.  FYI, though, in the future we'll probably make use of this and other new language features in R 4.1 and 4.2,  and make those as hard requirements in DESCRIPTION.

**Comment by andtheWings** [comment]
It is working now! 

I've been meaning to switch to the CRAN-sponsored apt repos. They keep much more fresh than the Ubuntu-sponsored ones. This was good motivation to do so once I verified the bug-fix! Now at R 4.1.3. :smile:
